ambry-router/src/main/java/com.github.ambry.router/PutOperation.java

```diff
@@ -179,12 +179,12 @@ void startReadingFromChannel() {
       public void onCompletion(Long result, Exception exception) {
         if (exception != null) {
           setOperationExceptionAndComplete(exception);
+          routerCallback.onPollReady();
         } else {
           blobSize = result;
           chunkFillingCompletedSuccessfully = true;
         }
         chunkFillerChannel.close();
-        routerCallback.onPollReady();
```

Comment:
So this onPollReady() was removed from this location because it will be called anyway in `fillChunks()`?